.github/workflows/checks.yml

```diff
@@ -0,0 +1,39 @@
+name: "PR Checks"
+
+on: [pull_request]
+
+jobs:
+  check_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.6
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Restore yarn cache
+        uses: actions/cache@v4
+        with:
+          path: .yarn/cache
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Grant read access to private registry
+        run: yarn config set --home 'npmRegistries["//registry.npmjs.org/"].npmAuthToken' "${{ secrets.NPM_TOKEN }}"
```

Comment:
Not sure what should be the value of `NPM_TOKEN`, do we have a shared one, or should I create one with my credentials?

Reply:
it is automatically available on private repos within the org, but needs to be made available to forks. @RyanZim can you help with that?

Reply:
Shouldn't be needed after https://github.com/ExodusMovement/eslint-config/pull/73#issuecomment-2226013816 is fixed.